templates/server/puppetserver/conf.d/puppetserver.conf.erb

```diff
@@ -73,6 +73,20 @@ jruby-puppet: {
 
     # (optional) enable or disable environment class cache
     environment-class-cache-enabled: <%= @server_environment_class_cache_enabled %>
+
+    # (optional) A map of environment variables which are made visible to
+    # Ruby code running within JRuby, for example, via the Ruby ENV class.
+    # By default, the only environment variables whose values are set into JRuby from the shell are HOME and PATH.
+    # The default value for the GEM_HOME environment variable in JRuby is set from the value provided for the
+    # jruby-puppet.gem-home key.
+    # Any variable set from the map for the environment-vars key overrides these defaults.
+    # Avoid overriding HOME, PATH, or GEM_HOME here because these values are already configurable via the shell
+    # or jruby-puppet.gem-home.
+    environment-vars: {
+    <%- @server_environment_vars.each do |env_key, env_val| -%>
+      "<%= env_key %>" : <%= env_val %>
```

Comment:
Is it needed to quote the value?

Reply:
The more I read about HOCON the more complicated it gets. My use case is to set the following
```hocon
  ...
  environment-vars: { 
    "FOO" : ${FOO}
  }
  ...
```
`${FOO}` will be replaced by the corresponding environment variable making it possible to pass through variables set in `/etc/default/puppetserver`. Quoting it in any way prevents substitutions from being evaluated.
So I guess we somehow must let the user decide whether to quote the value or not. So how about this:

We render the value verbatim into the config and adapt the docs to show that the following is possible:
```yaml
puppet::server_environment_vars:
  myvar: 'some string'
  myvar_from_env: '${myvar_from_env}'
  myvar_quoted: '"somestring with forbidden chars $[]{}"'
  myvar_multiline: |-
    """this
    is
    a
    multi-line
    string
    """
````
Would that be ok?

Reply:
I finally found the time to improve the variable doc comment. Can you take a look?

Reply:
@ekohl Any update on this?

Reply:
@ekohl Can you take another look?